control/control.go

```diff
@@ -287,7 +304,7 @@ func (c *Controller) Solve(ctx context.Context, req *controlapi.SolveRequest) (*
 		return nil, err
 	}
 	return &controlapi.SolveResponse{
-		ExporterResponse: resp.ExporterResponse,
+		ExportersResponse: resp.ExportersResponse,
```

Comment:
Need to set old field as well with the result of the first element.

Reply:
done.